builders/build/substrate/customizing-chain-specs.md

```diff
@@ -0,0 +1,192 @@
+- **Parachain Id** - sets the parachain id that has been reserved and assigned in the relay chain
+- **Code Substitutes** - an emergency feature to replace the runtime when an Appchain is unable to perform a runtime upgrade
+- **Properties** - key-value properties that can be customized and are useful to improve the user experience
+
+In the `properties` attribute, the following settings are used by various front-end libraries, including the [Polkadot.js API](/builders/interact/substrate-api/polkadot-js-api){target=_blank}:
+
+- **Token Symbol** - a name for your ContainerChain's own token symbol
+- **SS58 Format** - a unique integer that uniquely identifies the accounts in your network with [SS58 encoding](https://docs.substrate.io/reference/address-formats/){target=_blank}. All Substrated-based accounts have the same 32 bytes representation, that are base-58 encoded using this integer, which should be unique between networks
+- **Token Decimals** - represent how divisible a token can be, and what is the smallest representation of the token
+- **Is Ethereum** - a boolean identifying the network as EVM compatible or not
+
+## The Genesis State {: #genesis-state }
+
+All the collators assigned to the ContainerChain must agree on the initial state so they can execute the incoming extrinsics, arrive at the same results, and finally reach a consensus on the new valid state.
+
+This genesis state will define the starting point of the ContainerChain. It includes an initial value for the elements that the modules included in the runtime need to persist and the initial runtime Wasm code, which is stored on-chain.
+
+For example, in the templates included, the chain specification defines the initial accounts and token balances in the `Balances` module. In addition, the template also has a sudo account (which **should be modified**) for the `Sudo` module, which provides unique priviledges to the given account, and that can be offboarded once an on-chain democracy module is plugged in.
+
+## Generating a JSON Chain Specification File {: #generating-json-chain-specs }
+
+The following commands will build and generate the chain specification for the EVM-compatible template based on the configuration expressed in the `chain_spec.rs`, located in the `*/container-chains/templates/frontier/node/src/chain_spec.rs`. This example can be adapted to any other template or custom runtime.
+
+To build and generate the chain specifications, take the following steps:
+
+1. Clone the Tanssi code hosted on GitHub
+```bash
+git clone https://github.com/moondance-labs/tanssi
+```
+2. Step into the project folder
+```bash
+cd tanssi
+```
+3. Build the EVM-compatible Appchain template
+```bash
+cargo build -p container-chain-template-frontier-node --release
+```
+4. Generate the chain specification
+```bash
+./target/release/container-chain-template-frontier-node \
+    build-spec > chain_spec.json
+```
+
```

Comment:
Add here what happens after you generate the spec file, like where is the file going to be saved? - Normally it would be great to also show a terminal line screenshot showcasing the logs

Also remind the reader that this spits both the spec file and the genesis state

Reply:
Also I feel that it is important that you say here or somewhere that for Tnassi what we need is the raw spec file, so you can't just follow this and call it a day. Maybe with a `!!! note` tag or something in bold

Reply:
I'm still missing the screenshot, it will go separated in another PR